src/PlanView/FWLandingPatternMapVisual.qml

```diff
@@ -178,6 +179,7 @@ Item {
         MouseArea {
             anchors.fill:   map
             z:              QGroundControl.zOrderMapItems + 1   // Over item indicators
+            enabled:        _root.interactive
```

Comment:
I'd change this to visible which would seem safer. I think they end up being the same thing. But Qml is quirky as hell so who knows.